connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -168,12 +169,13 @@ public void setUp() throws Exception {
         EasyMock.expect(worker.isSinkConnector(CONN1)).andStubReturn(Boolean.FALSE);
         time = new MockTime();
 
-        herder = PowerMock.createPartialMock(DistributedHerder.class, new String[]{"backoff", "updateDeletedConnectorStatus"},
+        herder = PowerMock.createPartialMock(DistributedHerder.class, new String[]{"backoff", "connectorType", "updateDeletedConnectorStatus"},
```

Comment:
This is presumably why the tests didn't catch the issue -- this is mocked in all the tests, so we never validate class name in -> connector type out since the signature doesn't guarantee correctness via the type system since connector names and class names are both just `String`s.